core/backend/src/IModelDb.ts

```diff
@@ -364,8 +364,8 @@ export abstract class IModelDb extends IModel {
     }
   }
 
-  /** Close this IModel, if it is currently open. */
-  public close(): void {
+  /** Close this IModel, if it is currently open, and save changes if it was opened in ReadWrite mode. */
+  public close(description?: string): void {
```

Comment:
It's unclear to me who would ever call `close` with a description - the only reason to do that would be if you know the file contains unsaved changes and what those changes are, in which case, why aren't you calling `saveChanges`?